types.ts

```diff
@@ -163,15 +163,6 @@ export type SelfcareParty = {
   origin: string;
 };
 
-export type Party = {
```

Comment:
restore this

Reply:
You are changing Party and SelfcareParty models! Recover them from previous commit

Reply:
this has not been resolved